src/DeterministicNoiseSimulator.cpp

```diff
@@ -470,36 +538,18 @@ std::map<std::string, double> DeterministicNoiseSimulator::AnalyseState(dd::Qubi
             }
             current_result = current_result >> 1;
         }
-        measure_result.insert({result_string, global_probability});
+        if (global_probability > 0.01) {
```

Comment:
`0.01` seems like a magic number to me. shouldn't this probably be a parameter (with a default value of `0.01`)?

Reply:
Indeed. I added such a variable:
> double measurementThreshold = 0.01;